CHANGELOG.md

```diff
@@ -10,6 +26,10 @@ All notable changes to this project will be documented in this file.
 
 - \(PA-4552\) Create ci:test:setup step for iterative workflow [\#194](https://github.com/voxpupuli/beaker-puppet/pull/194) ([joshcooper](https://github.com/joshcooper))
 
+**Merged pull requests:**
+
+- Release 1.28.0 [\#201](https://github.com/voxpupuli/beaker-puppet/pull/201) ([joshcooper](https://github.com/joshcooper))
```

Comment:
Normally we skip these. I've added the skip-changelog label now so it should be gone if you regenerate it.

Reply:
Hm it seems that running `bundle exec rake changelog` didn't change anything, so I added a github changelog generator file to ignore that label if you think that's appropriate? (After rerunning the task it removed the lines you mentioned)

Reply:
because the changelog generator caches in /tmp/something. We exclude the label in the Rakefile:
```
config.exclude_labels = %w{duplicate question invalid wontfix wont-fix skip-changelog}
```

Reply:
Gotcha, weird not sure why it still picking up that PR earlier then ¯\_(ツ)_/¯